CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# How Can I Contribute?
+
+You can contribute to the project basically by fixing bugs or by suggesting enhancements.
+
+## Reporting bugs
+
+This section guides you through submitting a bug report for ObsRailsTemplate. Following these guidelines helps maintainers and the community understand your report, reproduce the behaviour, and find related reports.
```

Comment:
I couldn't find a way to place "variables" that will get replaced with the final project name once this template is used to create a new project.

Reply:
I also did a quick search and found no solution.